ppac-service-controller/mvp/src/main/kotlin/com/levi9/ppac/service/api/service/controller/mvp/CodesController.kt

```diff
@@ -51,6 +50,30 @@ class CodesController(
         return ResponseEntity(HttpStatus.UNAUTHORIZED)
     }
 
+    @Operation(
+        summary = "Check if user has company rights",
+        description = "Returns UNAUTHORIZED if the code received and codeId doesn't match"
+    )
+    @ApiResponses(
+        value = [ApiResponse(responseCode = "200", description = "OK"), ApiResponse(
+            responseCode = "401",
+            description = "Unauthorized"
+        )]
+    )
+    @GetMapping("/checkCompanyCode/{codeId}")
```

Comment:
what name is **codeId**? please rename to **companyId**, cause you check the combination AccessCode (from header) and companyId from path; you check to see if an access code is valid for a company given by companyId